### ISSUE TITLE
Support for multiple fields on the same page

### ISSUE THREAD

**Issue by edgaru** [Simon-the-Shark/django-mapbox-location-field#13]
```python
class Order(BaseModel):
    starting_location = LocationField()
    destination_location = LocationField()
```
The two fields refer to the same map.

What about add container to map_attrs and generate multiple instances using a class selector?
```javascript
var map = new mapboxgl.Map({
            container: 'secret-id-map-mapbox-location-field',
            style: map_attr_style,
            center: map_attr_center,
            zoom: map_attr_zoom,
});
```

**Comment by Simon-the-Shark** [comment]
I merged your pull request, but it's not a complete solution (more details in this pull request), so I'll be working on it and you can expect this feature to be implemented in the next few days.

**Comment by Simon-the-Shark** [comment]
Done! It's now available in version 1.6.0. It was a pleasure to cooperate with you. Feel free to open new issue whenever you encounter any problems or bugs (especially with this new feature). I hope the final result is satisfying for you.
